R/ckan_fetch.R

```diff
@@ -50,6 +50,12 @@
 #' x <- ckan_fetch(res$url)
 #' class(x)
 #' plot(x)
+#'
+#' GeoJSON file - requires geojsonio package
```

Comment:
make sure to comment out comments in the examples

Reply:
oops! thank you!